### PR TITLE
optimize util.crc32

### PR DESCRIPTION
**Pull request by ofek**

### CODE REVIEW

**Comment by dpkp** [comment]
Apologies -- all test fixtures were failing due to an unrelated issue. Please pull latest changes from master and re-run tests.

File: kafka/util.py [hunk 1]
```diff
@@ -12,14 +12,21 @@
...
+    def crc32(data):
+        crc = binascii.crc32(data)
+        # py2 and py3 behave a little differently
+        # CRC is encoded as a signed int in kafka protocol
+        # so we'll convert the py3 unsigned result to signed
+        if crc >= MAX_INT:
+            crc -= TO_SIGNED
+        return crc
+else:
+    def crc32(data):
```
Comment (dpkp) [inline kafka/util.py]: this could also just be `from binascii import crc32`

**Comment by dpkp** [comment]
Thanks!

File: kafka/util.py (hunk 1 above)
Comment (jeffwidman) [reply kafka/util.py]: Cleaned up in https://github.com/dpkp/kafka-python/commit/04fb37f690df5d887f9f191dc4d07408307c8207

**Comment by ofek** [comment]
Thank you both!

**Pull request merged**